exec/java-exec/src/main/java/org/apache/drill/exec/store/mock/MockStorageEngineConfig.java

```diff
@@ -34,6 +34,7 @@
   @JsonCreator
   public MockStorageEngineConfig(@JsonProperty("url") String url) {
     this.url = url;
+    this.setEnabled(true);
```

Comment:
This is not correct place to enable plugin config. You enable plugin after its creation, not directly in config itself.